src/main/java/com/powsybl/openloadflow/sa/DcSecurityAnalysis.java

```diff
@@ -238,144 +63,160 @@ private List<OperatorStrategyResult> createOperatorStrategyResults(DcSecurityAna
         LfTopoConfig topoConfig = new LfTopoConfig();
         findAllSwitchesToOperate(network, actions, topoConfig);
 
-        // try to find all pst to retain because involved in pst actions
+        // try to find all ptc to retain because involved in ptc actions
         findAllPtcToOperate(actions, topoConfig);
```

Comment:
@Hadrien-Godard indeed maybe we have to add RTC to operate if DC security analysis is used as a fall back. I think that actions are not going to be filtered and we can fall in issues.